File: CoffeeShopManager.API/Models/Staff.cs
using System;
using System.Collections.Generic;

namespace CoffeeShopManager.API.Models
{
    public class Staff
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public Team Team { get; set; }

        public int TeamId { get; set; }

       public string Gender { get; set; }

        public DateTime DateOfBirth { get; set; }

        public string Phone { get; set; }

        public string Email { get; set; }

        public string Address { get; set; }

        public string Photo { get; set; }

        public ICollection<Photo> Photos { get; set; }
    }
}

File: CoffeeShopManager-SPA/src/app/_models/Staff.ts
import { Photo } from './Photo';

export interface Staff {
    id: number;
    name: string;
    teamId: number;
    age: number;
    photo: string;
    gender: string;
    phone: string;
    email: string;
    address: string;
    dateOfBirth: Date;
    isDelete: boolean;
    photos?: Photo[];
}
